apps/metadata-editor/src/app/edit/edit-page.component.ts

```diff
@@ -137,18 +137,6 @@ export class EditPageComponent implements OnInit, OnDestroy {
         )
       })
     )
-    // if the record unique identifier changes, navigate to /edit/newUuid
-    this.facade.record$
-      .pipe(
-        filter(
-          (record) =>
-            record?.uniqueIdentifier !== currentRecord.uniqueIdentifier
-        ),
-        take(1)
-      )
-      .subscribe((savedRecord) => {
-        this.router.navigate(['edit', savedRecord.uniqueIdentifier])
-      })
```

Comment:
This is actually needed because when a record is first saved its ID is replaced from a temporary one (TEMP-ID-xxxx) to a final UUID